### PR TITLE
Retry create-dbs with sleeps

### PR DESCRIPTION
**Pull request by Gluestick**
Tests are still failing sometimes with the latest retry changes, so let's add some sleeps and more retries and see if that works.

### CODE REVIEW

File: create-dbs.clj [hunk 1]
```diff
@@ -24,7 +26,7 @@
                    (set))]
   ;; Datomic may not yet have fully started at this point, so we must retry a couple of times.
   ;; We fetch database names to ensure datomic is available, not because it's actually useful to do so.
-  (retry 5 d/get-database-names "datomic:dev://localhost:4334/*?password=123")
+  (retry 15 1000 d/get-database-names "datomic:dev://localhost:4334/*?password=123")
```
Comment (thumbnail) [inline create-dbs.clj:29]: image

**Comment by Gluestick** [comment]
So the first few attempts didn't seem to have the indended effect. Currently it retries the actual database creation with timeouts inbetween each retry, and this has had a 100% success rate so far (across several test runs in both grid/flex/staff).

TL;DR I'm confident it now works as intended.
